composer.json

```diff
@@ -33,15 +33,14 @@
         "symfony/dependency-injection": "^2.8.3 || ^3.0.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.1",
         "phpunit/php-token-stream": "^1.1.8",
         "php-http/curl-client": "^1.0",
         "php-http/socket-client": "^1.0",
         "php-http/guzzle6-adapter": "^1.1.1",
         "php-http/react-adapter": "^0.2.1",
         "php-http/buzz-adapter": "^0.3",
         "php-http/mock-client": "^1.0",
-        "symfony/phpunit-bridge": "^3.2",
+        "symfony/phpunit-bridge": "^3.4@rc || ^4.0",
```

Comment:
3.4 will allow PHPunit6.

Reply:
that should be possible with older versions of the bridge too

Reply:
Hm.. I maybe got confused by this PR https://github.com/symfony/symfony/pull/23952

Reply:
That was merged into the `3.3` branch (I forgot to update the changelog in the PR but that was fixed after merging). `^3.3 || ^4.0` should be good here.